STM32Cube/Tasks/trajectory.h

```diff
@@ -15,27 +15,13 @@ extern xQueueHandle altQueue;
 extern xQueueHandle angleQueue;
 extern xQueueHandle extQueue;
 
-void trajectory_task(void * argument);
+void trajectory_task(void* argument);
 void trajectory_init(void);
 
-//Struct with the data iterated in RK4 method
-typedef struct  RK4StateStruct{
-    float velY; //m/s
-    float velX; //m/s
-    float alt; //m
-} RK4State;
-
-//Struct with the forces acting on the rocket
-typedef struct ForceStruct{
-    float Fy; //N
-    float Fx; //N
-} Forces;
-
-//Struct for recieving altitude data, could replace with import
-typedef struct AltStruct{
-    int32_t alt; //altitude in m
-    float time; //time in ms
+// Struct for recieving altitude data, could replace with import
```

Comment:
could repl w/ what import? no clue what this is referring to but this struct shld prob be in traj lib with all the other data structs

Reply:
This was already there, don't know what it means